$OpenBSD: patch-configure,v 1.9 2012/01/09 12:12:44 dcoppa Exp $

Use '-pthread' instead of '-lpthread' for libpthread linkage

Remove -ldl

Fix GECKO_CFLAGS and GECKO_LIBS

--- configure.orig	Thu Dec 29 15:32:00 2011
+++ configure	Mon Jan  9 12:42:11 2012
@@ -7352,8 +7352,8 @@ fi
 if test "${with_xulrunner_sdk+set}" = set; then :
   withval=$with_xulrunner_sdk;
 	XULRUNNER_SDK_PREFIX=$with_xulrunner_sdk
-	GECKO_CFLAGS="-I$XULRUNNER_SDK_PREFIX/include -I$XULRUNNER_SDK_PREFIX/include/plugin -I$XULRUNNER_SDK_PREFIX/include/nspr -I$XULRUNNER_SDK_PREFIX/include/xpcom -I$XULRUNNER_SDK_PREFIX/include/string -I$XULRUNNER_SDK_PREFIX/include/java"
-	GECKO_LIBS="-L$XULRUNNER_SDK_PREFIX/lib -lxpcomglue_s -lxul -lxpcom -lplds4 -lplc4 -lnspr4 -lpthread -ldl"
+	GECKO_CFLAGS="-I$XULRUNNER_SDK_PREFIX/include -DOPENBSD"
+	GECKO_LIBS="-L$XULRUNNER_SDK_PREFIX -lxpcomglue_s -lxul -lxpcom -lplds4 -lplc4 -lnspr4 -pthread"
 	# Check whether --enable-new-libxul was given.
 if test "${enable_new_libxul+set}" = set; then :
   enableval=$enable_new_libxul; usenewlibxul=$enableval
